### ISSUE TITLE
Running streamspray example notebook for Orbit initialized by SkyCoord

### ISSUE THREAD

**Issue by nstarman** [jobovy/streamtools#2]
I ran https://streamtools.readthedocs.io/en/latest/notebooks/streamspray_example.html, but initialized the orbit with a SkyCoord, not list of floats. The function `streamspraydf.sample` throws a unit conversion error because `ro` is a quantity, but the internals of `streamspray.rtide` strips the units from other distances.

**Comment by jobovy** [comment]
Yeah, ``streamspraydf`` does not fully support ``SkyCoord``s and ``Quantity`` inputs. PR would be welcomed (but we also need some unit tests of ``streamspraydf`` first...)

**Comment by nstarman** [comment]
It could just be a decorator that does `.to_value(<unit>)` on specified inputs. That might be an easy PR ?
